File: shoppingm8_fe/lib/common/dto/errorDto.dart
class ErrorDto {
  final num code;
  final String message;
  final String uri;

  ErrorDto({this.code, this.message, this.uri});

  factory ErrorDto.fromJson(Map<String, dynamic> json) {
    return ErrorDto(
      code: json['code'],
      message: json['message'],
      uri: json['uri']
    );
  }
}

File: ShoppingM8-be/src/main/java/edu/pw/shoppingm8/base/api/model/ErrorDto.java
package edu.pw.shoppingm8.base.api.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ErrorDto {
    private int code;
    private String message;
    private String uri;
}
